yt_dlp/extractor/mediaset.py

```diff
@@ -218,6 +294,29 @@ def _real_extract(self, url):
         return info
 
 
+class MediasetClipIE(MediasetIE):
+    _VALID_URL = r'''(?x)
+                     https?://
+                         (?:www\.)?\w+\.mediaset\.it/
+                             video/[\w-]+_(?P<id>\d{5,})\.s?html?
+                 '''
```

Comment:
```suggestion
    _VALID_URL = r'(?x)https?://(?:www\.)?\w+\.mediaset\.it/video/[\w-]+_(?P<id>\d{5,})\.s?html?'
```

Reply:
Then you can take out `(?x)` too, no?

Reply:
Right, my bad. I was originally just trying to fix the indents, but then decided it's too small to be worth splitting into lines

Reply:
I left the spacing in case anyone had more patterns to add but it's fine to compress it.